src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={classNames(
+              selectedTab.title === tab.title && 'is-active',
+            )}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => (selectedTab !== tab && onTabSelected(tab))}
```

Comment:
I would recommend to compare tabs using their `id` to avoid possible conflicts when comparing